fenics_ice/model.py

```diff
@@ -162,8 +162,8 @@ def init_fields_from_data(self):
 
         if self.params.melt.use_melt_parameterisation:
        
-         melt_depth_therm_const: float = -999.0
-         melt_max_const: float = -999.0
+         #melt_depth_therm_const: float = -999.0
+         #melt_max_const: float = -999.0
```

Comment:
Is this change intended?

Reply:
@jrmaddison yes it is -- i think i copied and pasted from config.py to remind myself of param names and forgot to delete them. 

these parameters are only referred to below using the self.params object -- and so are not (i don't believe) used.

Reply:
OK, I guess these lines can be deleted?

Reply:
will do